openff/toolkit/utils/openeye_wrapper.py

```diff
@@ -840,13 +842,14 @@ def enumerate_protomers(
 
```

Comment:
I wish GitHub would let you leave suggestions anywhere.

In line 835/837:

```diff
- # add one as the input is included
- options.SetMaxCount(max_states + 1)
+ options.SetMaxCount(max_states)
```

The original behavior was to ask for n+1 protomers, and then throw one away to end up with n. Since we now return the original, we just want to ask for n. Incidentally this means the original behavior was subtly wrong, as if OpenEye did not return the input molecule we would give n+1 molecules back. It also meant there was no way to ask for all protomers (`SetMaxCount(0)`).